integration/revalidate-test.ts

```diff
@@ -44,6 +44,11 @@ test.describe("Revalidation", () => {
             }
           `,
 
+          "app/routes/_index.jsx": js`
+            export default function Component() {
+              return <h1>Index</h1>;
+            }
+          `,
```

Comment:
Needed an index route in this existing test.  Putting `path:""` on the root previously meant that `/` would match the root route without it having an `index.jsx` or `$.jsx` file, which seems like a related bug that's also fixed here